deploy/streisand-new-cloud-server

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+#
+# Run a noninteractive Streisand installation that creates a new cloud server.
+#
+# This requires an expanded extra-vars file specific to the provider type that
+# sets all of the values gathered by prompts in the interactive installation.
+# See the contents of global_vars/noninteractive for examples that can be copied
+# and modified.
+#
+# Usage:
+# streisand-new-cloud-server \
+#   --provider digitalocean \
```

Comment:
It might make sense to express this usage like:
`--provider [amazon|azure|digitalocean|google|linode|rackspace]` to make it clearer that you can use those additional providers.